### PR TITLE
params: delete asset task params

### PR DESCRIPTION
**Pull request by gioelecerati**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -315,6 +316,11 @@ type (
 		CatalystPipelineStrategy string    `json:"catalystPipelineStrategy,omitempty"`
 	}
 
+	DeleteAssetTaskParams struct {
+		Id            string `json:"id"`
```
Comment (victorges) [inline api.go:320]: ~~Can you specify what is this id? I assume it's not the task id.~~

Actually, I don't think you need to specify. You can just remove all these fields. The input (and if relevant, output) assets are already specified in the root of the task object and should not be repeated in the params, as that'd be redundant and lead to code repetition (like not reusing the existing logic that fetches all the task context (check comment below)).

File: api.go [hunk 2]
```diff
@@ -315,6 +316,11 @@ type (
...
+		ObjectStoreId string `json:"objectStoreId"`
```
Comment (victorges) [inline api.go:321]: You don't need this! The task should already have an `inputAssetId` field, which `task/runner.go` will already make sure to fetch the full object, grab the object store id from that object, fetch the object store and instantiate the client for your task handler. Check the task context

**Review by victorges — CHANGES_REQUESTED** [review]
I don't think we need any custom params for the delete task! `inputAssetId` is the only thing you will need to set from the API. Creating separate params will only create code repetition.

File: api.go (hunk 2 above)
Comment (gioelecerati) [reply api.go:321]: Yeah I noticed when editing the task runner! Will update that :)

**Comment by gioelecerati** [comment]
Closing this as params are not needed

**Pull request closed without merging**